content/guides/hosting/data-security/presigned-urls.md

```diff
@@ -50,4 +50,10 @@ Pre-signed URLs are the only supported blob storage access mechanism in W&B. W&B
 {{% /alert %}}
 
 ### Determine the user that requested a pre-signed URL
-When W&B returns a pre-signed URL for AWS or GCP blob storage, the `X-User` header contains the requester's username. The header is not set for Azure blob storage.
\ No newline at end of file
+When W&B returns a pre-signed URL, the response includes the requester's username:
+
+| Storage provider   | Response field  |
```

Comment:
It's not the response from W&B that has the `X-User` header, it's a query parameter of the signed URL that W&B returns. That signed URL then is used by the client to perform storage operations. That's the case for AWS and GCP

Something like this:
```suggestion
| Storage provider   | Signed URL Query Parameter |
```